R/utilities-figures.R

```diff
@@ -252,29 +252,26 @@ createPlotsFromExcel <- function(simulatedScenarios, observedData, projectConfig
   # read sheet "exportConfiguration"
   dfExportConfigurations <- readExcel(file.path(projectConfiguration$paramsFolder, projectConfiguration$plotsFile), sheet = "exportConfiguration") %>%
     rename(name = outputName)
-  dfDataCombined <- .validateDataCombinedFromExcel(dfDataCombined, simulatedScenarios, observedData, stopIfNotFound)
-  dataCombinedNames <- unique(dfDataCombined$DataCombinedName)
-  dfPlotConfigurations <- .validatePlotConfigurationFromExcel(dfPlotConfigurations, dataCombinedNames)
+
   dfPlotGrids <- .validatePlotGridsFromExcel(dfPlotGrids, unique(dfPlotConfigurations$plotID))
+  # Filter and validate only used plot configurations
```

Comment:
Inverted the validation sequence and directly removing unused `DataCombined` and `PlotGrids` so they are not validated if not used.